gromacs/__init__.py

```diff
@@ -242,6 +242,15 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+# Try to load environment variables setted by GMXRC
+try:
+    import os
+    if 'GMXPREFIX' not in os.environ:
```

Comment:
`GMXPREFIX` only works for Gromacs 5.x; we don't have it for Gromacs 4.6.x. Check for the presence of `GMXBIN` instead (or first `GMXPREFIX` and then `GMXBIN`).